projects/cashmere/src/lib/file-size-pipe/file-size.pipe.ts

```diff
@@ -0,0 +1,44 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+// Convert bytes into largest possible unit.
+// Credit to https://gist.github.com/JonCatmull/ecdf9441aaa37336d9ae2c7f9cb7289a
+@Pipe({ 'name': 'fileSize' })
```

Comment:
This should be a `pure: true` pipe, right?

Reply:
Pipes are pure by default.
https://angular.io/guide/pipes#pure-and-impure-pipes

Reply:
true that :)